examples/cart/index.js

```diff
@@ -9,9 +9,17 @@ $(function() {
   });
 
   var cart;
-  client.createCart().then(function (newCart) {
-    cart = newCart;
-  });
+  if(localStorage.lastCartId) {
```

Comment:
use `getItem` and `setItem`. If we add tests to this, those can be stubbed. Testing that properties are accessed is harder.